src/Resources/npm-package/js/contao-multi-column-editor-bundle.js

```diff
@@ -198,6 +198,10 @@ class MultiColumnEditorBundle {
                         MultiColumnEditorBundle.initChosen(widget);
                         MultiColumnEditorBundle.initSortable(isBackend);
 
+                        MultiColumnEditorBundle.hideTooltip();
+                        setTimeout(function () {MultiColumnEditorBundle.hideTooltip();}, 1000); // hide delayed tooltips
```

Comment:
It's not a nice solution, but I don't know how else to fix it 😪

Reply:
It worked for me and if there is no other solution, it's ok for me. What do think @Defcon0 ?

Reply:
Not nice, but if it works, I'm ok with it. Thanks @rabaus !

Reply:
It's even better than the current state with never hiding tooltips ;-)
Maybe we will find a better solution while implementing a PR for the related issue in the contao-core 😉

Reply:
Sure, let us know :)